packages/main/src/components/Breadcrumbs/index.tsx

```diff
@@ -30,35 +42,127 @@ export interface BreadcrumbsPropTypes extends CommonProps {
   currentLocationText?: string;
 }
 
+const useStyles = createUseStyles(
+  {
+    list: {
+      listStyleType: 'none',
+      display: 'flex',
+      alignItems: 'center',
+      marginBlock: 0,
+      paddingInline: 0,
+      flexWrap: 'wrap',
+      '& li': {
+        display: 'flex',
+        overflow: 'hidden'
+      }
+    },
+    separatorInline: {
+      margin: '0 0.25rem'
+    },
+    currentLocation: {
+      fontFamily: ThemingParameters.sapFontFamily,
+      fontSize: ThemingParameters.sapFontSize,
+      color: ThemingParameters.sapContent_LabelColor,
+      whiteSpace: 'nowrap',
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
+      '&:focus': {
+        outline: `${ThemingParameters.sapContent_FocusWidth} ${ThemingParameters.sapContent_FocusStyle} ${ThemingParameters.sapContent_FocusColor}`,
+        outlineOffset: '-0.0625rem'
```

Comment:
shouldn't the offset be the same as the outline width? 
```suggestion
        outlineOffset: ThemingParameters.sapContent_FocusWidth
```